infra/frontend/app-config/env-config/environment-variables.tf

```diff
@@ -7,6 +7,8 @@ locals {
     # WORKER_THREADS_COUNT    = 4
     # LOG_LEVEL               = "info"
     # DB_CONNECTION_POOL_SIZE = 5
+    NEW_RELIC_ENABLED = false
```

Comment:
Docs link please! Probably to here:

https://docs.newrelic.com/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#agent-enabled